classes/ValidFormBuilder/MultiField.php

```diff
@@ -298,6 +295,20 @@ public function __toHtml($submitted = false, $blnSimpleLayout = false, $blnLabel
 
         $this->setMeta("class", "vf__multifield");
 
+        if ($this->isRemovable()) {
+            $this->setMeta("class", "vf__removable");
+        }
+
+        //*** Add data-dynamic="original" or data-dynamic="clone" attributes to dynamic fields
```

Comment:
Please explain in short why you add the data attributes. There is no direct logic to derive the "why" from.